components/header/index.js

```diff
@@ -57,15 +57,20 @@ const rightNav = {
       label: "Donate",
       href: "https://numfocus.org/donate-to-nteract",
       prefetch: false
-    }
+    },
+    {
+      label: "Help",
+      href: "https://nteract.slack.com/",
```

Comment:
Can we replace this with the invite URI?

https://join.slack.com/t/nteract/shared_invite/zt-ijrf78gm-ZldhpD7dzWzALnZs0JwGVg

Unfortunately, we'll have to update this every 30 days.